.test_director

```diff
@@ -18,15 +23,15 @@ NEW_TESTS=( $(git diff --name-only $(git merge-base origin/master HEAD) | grep '
 # no new tests found, just run improved sanity test
 if [ ${#NEW_TESTS[@]} -eq 0 ]; then
     ansible-playbook --syntax-check tests/improved-sanity-test/main.yml
-    ansible-playbook -vi testnode, tests/improved-sanity-test/main.yml
+    ansible-playbook -vi testnode, tests/improved-sanity-test/main.yml $EXTRA_ARGS
```

Comment:
Minor: you can just do `"$@"` to directly pass through arguments. So then you wouldn't have to quote the args in the YAML, e.g.:

```yaml
tests:
  - ./test_directory --skip-tags kernel_avc_denied
```

Reply:
Oh word, so I don't even have to check for the existence of the args.